vmdb/app/models/container_group.rb

```diff
@@ -9,7 +9,7 @@ class ContainerGroup < ActiveRecord::Base
   has_many :containers, :dependent => :destroy
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, :class_name => CustomAttribute, :as => :resource, :conditions => {:section => "labels"}
+  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource
```

Comment:
Having the lambda in the middle is melting my mind.  Is this the proper way to do this...typically blocks are at the end, but this is no ordinary block.  @matthewd @tenderlove Thoughts?